deploy/manager/dev/10_service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
I guess all the changes to dev resources would belong to make update commit so I am not going through them twice on review